test/test_server.py

```diff
@@ -224,7 +224,7 @@ def test_command3(self, mock_make_server, mock_browser):
 """)
 
         server.serve(server_root)
-        mock_browser.assert_called_once_with('"C:\Program Files\Mozilla Firefox\firefox.exe" %s &')
```

Comment:
By the way, is there any protection against peculiar arguments for substitutions? On linux I strongly prefer something from execve(2) family that has each command line argument as a separate array element.

Path to firefox at least surrounded with quotes... Is it assumed that argument for the `%s` substitution is passed through `urlencode`, so it is safe?